tests/testlib/wakunode.nim

```diff
@@ -38,10 +40,26 @@ proc newTestWakuNode*(nodeKey: crypto.PrivateKey,
                       discv5UdpPort = none(Port),
                       agentString = none(string),
                       peerStoreCapacity = none(int)): WakuNode =
+  
+  var resolvedExtIp = extIp
+
+  # Update extPort to default value if it's missing and there's an extIp or a DNS domain 
+  let extPort = if (extIp.isSome() or dns4DomainName.isSome()) and
```

Comment:
Why do you need to do this?

Reply:
It is based on the following code:
https://github.com/waku-org/nwaku/blob/dc25057a2632baf487a9ae00533db291f2791dc6/apps/wakunode2/internal_config.nim#L48-L52

I copied the behavior we currently have to create a node to `newTestWakuNode` so what we create the test node under the same logic that we use to create production nodes.

If we provide a DNS address and no port to create a Node, we use the default port.
But in `newTestWakuNode` that logic didn't exist, and it attempted to create a node without an extIp provided and without having a default one set, so an exception was thrown